commit 2f9fd1768ae474e5b9998867880a2369c47d9916
Author: Eric Engestrom <eric@igalia.com>
Date:   Sat Sep 20 13:12:08 2025 +0200

    intel/meson: generate spirv_info.h before compiling brw_spirv.c
    
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/37544>

diff --git a/src/intel/compiler/elk/meson.build b/src/intel/compiler/elk/meson.build
index 07f91e243b3..fcf5e6b57d2 100644
--- a/src/intel/compiler/elk/meson.build
+++ b/src/intel/compiler/elk/meson.build
@@ -133,7 +133,7 @@ libintel_compiler_elk = static_library(
   c_args : [no_override_init_args],
   cpp_args : ['-Werror=vla'],
   gnu_symbol_visibility : 'hidden',
-  dependencies : [idep_nir_headers, idep_mesautil, idep_intel_dev],
+  dependencies : [idep_nir_headers, idep_mesautil, idep_intel_dev, idep_vtn],
   build_by_default : false,
 )
 
diff --git a/src/intel/compiler/meson.build b/src/intel/compiler/meson.build
index 2fff9740f6e..22e5c5d283d 100644
--- a/src/intel/compiler/meson.build
+++ b/src/intel/compiler/meson.build
@@ -159,7 +159,7 @@ libintel_compiler_brw = static_library(
   c_args : [no_override_init_args],
   cpp_args : ['-Werror=vla'],
   gnu_symbol_visibility : 'hidden',
-  dependencies : [idep_nir_headers, idep_mesautil, idep_intel_dev],
+  dependencies : [idep_nir_headers, idep_mesautil, idep_intel_dev, idep_vtn],
   build_by_default : false,
 )
 
